AirportTicketBooking/Utility/ConsoleOutput.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace AirportTicketBooking.Utility
+{
+    public static class ConsoleOutput
+    {
+        public static void NoDataFoundMessage(string dataName)
+        {
+            Console.WriteLine($"There are no {dataName} found!");
+        }
+
+        public static void ShowManagerDetails(int id, string name, string email)
+        {
+            Console.WriteLine($"{id}: {name} with the email {email}");
+        }
+        
+        public static void ShowPassengerDetails(int id, string name, string email)
+        {
+            Console.WriteLine($"{id}: {name} with the email {email}");
+        }
+
+        public static void PrintGeneralMessage(string message)
+        {
+            Console.WriteLine(message);
+        }
```

Comment:
I would like it to be more of returning the message as string instead of keep repeating the console.wrtieline() command everywhere

Reply:
Refactored.